post/admin.py

```
@@ -1 +1,8 @@
 from django.contrib import admin
+from django.contrib import admin
```

Ruff (0.11.9)

Redefinition of unused `admin` from line 1

Remove definition: `admin`

(F811)